.editorconfig

```diff
@@ -1,16 +0,0 @@
-# Editor configuration, see https://editorconfig.org
-root = true
-
-[*]
-charset = utf-8
-indent_style = space
-indent_size = 2
-insert_final_newline = true
-trim_trailing_whitespace = true
-max_line_length = off
-
-[*.ts]
-quote_type = single
-
-[*.md]
-trim_trailing_whitespace = false
```

Comment:
I think this file shouldn't be deleted since it's also on the svelte version. Is there a reason to delete it for the React version?

Reply:
I never had worked with .editorcong files before. so, I had no idea. what this file doing. after watching some videos from YouTube. Now, I know it is just inditation. This will be added in next commit